.github/actions/nm-set-env/action.yml

```diff
@@ -11,6 +11,7 @@ runs:
   using: composite
   steps:
   - run: |
+      echo "TORCH_CUDA_ARCH_LIST=7.0,7.5,8.0,8.6,8.9,9.0+PTX" >> $GITHUB_ENV
```

Comment:
the vllm Dockerfile separates these with spaces.  i'm not sure how strict the format is.

Reply:
seems like it might be fairly strict: https://github.com/pytorch/audio/issues/1940

Reply:
yeah, this was a typo. fixed.